.ci/aws/Jenkinsfile

```diff
@@ -134,6 +134,7 @@ pipeline {
     options {
         buildDiscarder(logRotator(daysToKeepStr: "90"))
         timeout(time: 24, unit: 'HOURS')
+        skipDefaultCheckout()
```

Comment:
Commit message and change doesn't align? Where do we define the next step to be workspace cleanup?

Reply:
> Where do we define the next step to be workspace cleanup?

The workspace clean up was removed in commit https://github.com/aws/aws-ofi-nccl/commit/a8009b82f50d3ba30d937970e8fe66838fe83a13. Also, in this commit, we removed

```
        stage("Checkout SCM repo") {
            steps {
                checkout scm
            }
        }
```

Based on Seth's explanation *"checkoutSCMRepo pulls down aws-ofi-nccl to the fargate agent, which is a waste of time because we only need it on the cluster."*

---

However, even though we explicitly removed the step above. Jenkins itself contains a default SCM checkout step, with this line of change, this step should be removed completely for us.

Reply:
Rev2: rebased changes

Reply:
Ack. Resolving conversation.